variables.tf

```
@@ -21,6 +22,11 @@ variable "event_rule" {
         input_paths    = optional(map(string))
         input_template = string
       }))
+
+      sqs_target = optional(object({
+        message_group_id = optional(string)
```

as written, message_group_id is required if sqs_target is provided...

```suggestion
        message_group_id = string
```